extensions/interactions/MultipleChoiceInput/directives/oppia-interactive-multiple-choice-input.component.ts

```diff
@@ -114,6 +146,14 @@ export class InteractiveMultipleChoiceInputComponent implements OnInit {
     // Selected current option.
     (event.currentTarget as HTMLDivElement).classList.add('selected');
     this.answer = parseInt(answer, 10);
+    // Store current choice order in question element to
+    // prevent shuffling of choices.
+    let questionElement = document.querySelector(
+      '.oppia-rte-viewer.oppia-learner-view-card-top-content');
+    if (questionElement) {
+      questionElement.setAttribute(
+        'choice-order', JSON.stringify(this.choices));
```

Comment:
Do we need to store all of the choices data here? Do these choices have a unique value like an ID that we can map them on?

Reply:
Yes, they have an original order index, modified this part to only store indexes instead of whole data